config/initializers/secure_headers.rb

```diff
@@ -9,8 +9,7 @@
   config.csp.merge!(
     default_src: %W(#{ENV['APP_PROTOCOL']}: 'self'),
     img_src: %w('self' https://maps.googleapis.com https://csi.gstatic.com https://*.s3.amazonaws.com data: https://maps.gstatic.com),
-    script_src: %w('self' https://maps.googleapis.com 'unsafe-eval'),
-    script_src: %w('self' https://js.stripe.com/v2/ 'unsafe-eval'),
+    script_src: %w('self' https://maps.googleapis.com https://js.stripe.com/v2/ ),
```

Comment:
Please double-check and make sure that all JS functionality works fine without `'unsafe-eval'` allowed here?